edgefirewall.yaml

```diff
@@ -549,90 +549,99 @@ components:
           type: boolean
 
     Behaviors:
+      oneOf:
+        - $ref: '#/components/schemas/NullArgumentBehavior'
+        - $ref: '#/components/schemas/SimpleArgumentBehavior'
+        - $ref: '#/components/schemas/SetRateLimitBehavior'
+        - $ref: '#/components/schemas/SetWAFRuleSetBehavior'
+
+    NullArgumentBehavior:
       type: object
       properties:
         name:
           type: string
-          enum: [deny, drop, set_rate_limit, set_waf_ruleset, run_function, tag_event]
+          enum: [deny, drop]
         argument:
-          oneOf:
-            - $ref: '#/components/schemas/GenericBehaviorDetails'
-            - $ref: '#/components/schemas/SetRateLimitDetails'
-            - $ref: '#/components/schemas/SetWAFRuleSetDetails'
-        
-      example:
-        behavior_example_1:
-          name: deny
-        behavior_example_2:
-          name: set_rate_limit
-          argument:
-            type: second
-            limit_by: client_ip
-            average_rate_limit: 100
-            maximum_burst_size: 10
-        behavior_example_3:
-          name: set_waf_ruleset
-          argument:
-            waf_id: 1
-            mode: Learning
-            
-    GenericBehaviorDetails:
+          type: integer
```

Comment:
Just out of curiosity... When this field may be an integer?

Reply:
As Null is not a valid type for Open API, this is the way defined their documentation to achieve this result.

<img width="821" alt="Captura de Tela 2023-10-16 às 09 10 11" src="https://github.com/aziontech/azionapi-openapi/assets/8815139/d4410286-410c-4984-b269-c21323e35c11">

Reply:
Got it, thanks for the explanation!